### PR TITLE
Fix 191

### LINKED ISSUE #191: Wine unload cross postings don’t get deleted by MAB
Wine trades via MAB dual post into the booze prep channel. We need to track those message ids and then ensure to clean them up.

likely needs some changes to format the message id into a list that MAB can iterate over,

### PR DESCRIPTION
**Pull request by cimspin**
Fixes #191

### CODE REVIEW

File: constants.py [hunk 1]
```diff
@@ -20,7 +20,7 @@
 
 TEST_DISCORD_GUILD = 818174236480897055 # test Discord server
 TEST_TRADE_ALERTS_ID = 843252609057423361  # trade alerts channel ID for PTN test server
-TEST_WINE_ALERTS_ID = 848492679079395371 # currently this ID is the testing channel in the test server
+TEST_WINE_ALERTS_ID = 870425638127943700 # currently this ID is the testing channel in the test server
```
Comment (gcruicks) [inline constants.py:23]: I see you fixed my lazy approach of pasting it all in one place

File: MissionAlertBot.py [hunk 2]
```diff
@@ -1253,12 +1253,17 @@ async def done(ctx, carrier_name_search_term, rp=None):
         # fill in some info for messages
         desc_msg = f"> {rp}\n\n" if rp else ""
 
+        # check if it's Wine, in which case it went to the booze cruise channel
+        if mission_data.commodity.title() == "Wine":
```
Comment (gcruicks) [inline MissionAlertBot.py:1257]: This should all probably be in the if block

**Review by gcruicks — DISMISSED** [review]
Doesn’t overly matter, but the channel check should be in the if block I think, else it’s pointless to check it out

File: MissionAlertBot.py (hunk 2 above)
Comment (cimspin) [reply MissionAlertBot.py:1257]: I moved it to the try: bit

**Review by gcruicks — APPROVED** [review]

**Pull request merged**